src/common/engines/openrouter.ts

```
@@ -4,96 +4,96 @@ import { AbstractEngine } from './abstract-engine'
+    let hasError = false
+    let finished = false
+    await fetchSSE(url, {
+      method: 'POST',
+      headers,
+      body: JSON.stringify(body),
+      signal: req.signal,
+      onMessage: async (msg) => {
+        if (finished) return
+        try {
+          const resp = JSON.parse(msg)
+          if (resp.choices && resp.choices[0].delta.content) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)